verification/utils/bitwise/bitwise-eqs.gobra

```diff
@@ -42,6 +42,13 @@ ensures res == b & 0x7
 decreases
 func BitAnd7(b int) (res int)
```

Comment:
this lemma is no longer necessary. Can you please drop it here and in the file `proofs.dfy`? In addition, can you please add a Dafny proof of lemma `BitAndF`? In addition, can you make its input/output type a `byte` instead of `int`?

Reply:
Done.